.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+deno.lock
```

Comment:
maybe this change is not related to this PR?

Reply:
This is because deno-gfm uses esm.sh for it's dependencies which can become outdated and cause this issue denoland/deno-gfm#47

Reply:
Hmm true. So we might want to "lock": false in deno.jsonc instead.

Reply:
Interesting, I did not know of that option in `deno.jsonc`